src/modules/addTodo.js

```diff
@@ -0,0 +1,44 @@
+import Task from './Task';
+import '../style.css';
+
+const input = document.getElementById('add-todo');
+const todosUL = document.getElementById('todos');
+const trash = document.getElementsByClassName('positioner');
+const checkBox = document.querySelectorAll('.checkbox');
+
+const todo = new Task();
+
+export default function addTodo() {
+  const todoText = input.value;
+  const todoEl = document.createElement('li');
+  todo.description = todoText;
+  todo.completed = checkBox.value;
+  todoEl.setAttribute('id', todosUL.childNodes.length);
+  todoEl.setAttribute('class', 'todo-el');
+  todoEl.innerHTML = `
+    <div>
+    <input 
+    type="checkbox" 
+    tabindex="0" 
+    class="checkbox" 
+    value="${todo.completed}"/>
+    <label for="checkbox">
+    <span class="todo-text">${todo.description}</span>
+    </label>
+    </div>
+    <button type="reset" class="positioner" >
+    <i tabindex="0" class="fa fa-trash-alt"></i>
+    </button> `;
+  todosUL.appendChild(todoEl);
+  input.value = '';
+  trash.attachEvent('reset', (e) => {
+    e.preventDefault();
+    todoEl.innerHTML = '';
+    localStorage.clear();
+  });
+  const todos = {};
+  localStorage.setItem(todos, JSON.stringify(todo));
+}
```

Comment:
You have implemented all the functionalities for creating the todo item correctly `addTodo()` however the code can be broken down into smaller functions that will allow you to reuse the same code without repeating yourself.

Reply:
Thanks for your feedback, but we supposed to create issues, no to do a review in the PR. Anyways, I will take your advice. Thanks again.